cfme/v2v/migrations.py

```diff
@@ -205,6 +226,10 @@ class InfraMappingWizardResultsView(View):
     close_btn = Button("Close")
     continue_to_plan_wizard_btn = Button("Continue to the plan wizard")
```

Comment:
close = Button("Close")
continue_to_plan_wizard = Button("Continue to the plan wizard")